src/Blockcore.Indexer.Core/Sync/SyncTasks/BlockIndexer.cs

```diff
@@ -294,15 +293,21 @@ await mongoData.Mempool.Indexes
          {
             if (indexingCompletTask is not { IsCompleted: true })
             {
-
                log.LogDebug($"Indexer - Indexing tables time passed {watch.Elapsed}");
             }
             else
             {
+               List<string> allIndexes = mongoData.GetAllIndexes().Where(w => w.Contains("BlockIndex")).ToList();
```

Comment:
Why do you filter only block index?
perhaps the name allIndexes should change to give a bit more information about that

Reply:
because I noticed that the total indexes will change for example I think the richlist creates a temp table then the index count is different, anyway reorgs happen on block index thats why I used it, its a fixed index count.

Reply:
> perhaps the name allIndexes should change to give a bit more information about that

Like what for example?
`GetAllIndexes` I could change the name to `GetAllBlockIndexProperties`?